integration-tests/jest/jest-utils.js

```diff
@@ -0,0 +1,21 @@
+const { screenshotDir } = require('../utils/integration-config')
+
+
+const defaultTimeout = 5 * 60 * 1000
```

Comment:
TOL: This timeout will need to apply in the web service as well, so it's not strictly a 'jest' thing. Not a big deal, but just wanted to call out a potential name confusion.

Reply:
True. The mechanism for implementing the timeout (giving it to jest vs `Promise.race`) may be different but the meaning is the same.

I don't love the options currently available. I'm thinking `integration-config.js`, or maybe `integration-helpers.js`. Any preference?

Reply:
`integration-helpers.js` seems like a decent place to me